tpcx_bb/queries/q03/tpcx_bb_query_03_sql.py

```diff
@@ -179,6 +184,9 @@ def main(data_dir, client, bc, config):
         LIMIT {q03_limit}
     """
     result = bc.sql(last_query)
+
+    bc.drop_table("item_df")
```

Comment:
move this ` bc.drop_table("item_df")` to after `wait(product_view_results)` and also do `del item_df`